tests/mclag_test.py

```diff
@@ -0,0 +1,665 @@
+import os
+import traceback
+
+from click.testing import CliRunner
+
+import config.main as config
+import show.main as show
+from utilities_common.db import Db
+
+
+
+MCLAG_DOMAIN_ID = "123"
+MCLAG_INVALID_DOMAIN_ID1 = "-1"
+MCLAG_INVALID_DOMAIN_ID2 = "5000"
+MCLAG_DOMAIN_ID2 = "500"
+MCLAG_DOMAIN_ID3 = "1000"
+MCLAG_SRC_IP    = "12.1.1.1"
+RESERVED_IP    = "0.0.0.0"
+INVALID_IP    = "255.255.255.255"
+NOT_IP    = "abcd"
+MCLAG_PEER_IP   = "12.1.1.2"
+MCLAG_KEEPALIVE_TIMER  = "5"
+MCLAG_SESSION_TIMEOUT  = "20"
+MCLAG_MEMBER_PO  = "PortChannel10"
+MCLAG_MEMBER_PO2 = "PortChannel20"
+MCLAG_UNIQUE_IP_VLAN  = "Vlan100"
+
+MCLAG_PEER_LINK = "PortChannel12"
+MCLAG_PEER_LINK2 = "PortChannel13"
+MCLAG_INVALID_SRC_IP1  = "12::1111"
+MCLAG_INVALID_SRC_IP2  = "224.1.1.1"
+MCLAG_INVALID_PEER_IP1  = "12::1112"
+MCLAG_INVALID_PEER_IP2  = "224.1.1.2"
+MCLAG_INVALID_PEER_LINK1  = "Eth1/3"
+MCLAG_INVALID_PEER_LINK2  = "Ethernet257"
+MCLAG_INVALID_PEER_LINK3  = "PortChannel123456"
+MCLAG_INVALID_PEER_LINK4  = "Lag111"
+MCLAG_INVALID_PEER_LINK5  = "Ethernet123456789"
+MCLAG_INVALID_KEEPALIVE_TIMER  = "11"
+MCLAG_INVALID_SESSION_TIMEOUT = "31"
+MCLAG_INVALID_KEEPALIVE_TIMER_LBOUND  = "0"
+MCLAG_INVALID_KEEPALIVE_TIMER_UBOUND  = "61"
+MCLAG_INVALID_SESSION_TMOUT_LBOUND  = "2"
+MCLAG_INVALID_SESSION_TMOUT_UBOUND  = "4000"
+
+MCLAG_INVALID_MCLAG_MEMBER  = "Ethernet4"
+MCLAG_INVALID_PORTCHANNEL1  = "portchannel" 
+MCLAG_INVALID_PORTCHANNEL2  = "PortChannelabcd" 
+MCLAG_INVALID_PORTCHANNEL3  = "PortChannel10000" 
+MCLAG_INVALID_PORTCHANNEL4  = "PortChannel00111" 
+
+
+MCLAG_UNIQUE_IP_INTF_INVALID1  = "Ethernet100"
+MCLAG_UNIQUE_IP_INTF_INVALID2  = "Ethernet100"
+
+class TestMclag(object):
+    @classmethod
+    def setup_class(cls):
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+        print("SETUP")
+
+    def verify_mclag_domain_cfg(self, db, domain_id, src_ip="", peer_ip="", peer_link=""):
+        keys = db.cfgdb.get_keys('MCLAG_DOMAIN')
+        key_found = False 
+        for k in keys:
+            if k == domain_id:
+                key_found = True
+        if key_found is False: 
+            return False 
+
+        mclag_entry = db.cfgdb.get_entry("MCLAG_DOMAIN", MCLAG_DOMAIN_ID)
+        if src_ip is not None:
+            temp = mclag_entry["source_ip"]
+            if temp is not None and temp != src_ip:
+                return False
+        if peer_ip is not None:
+            temp = mclag_entry["peer_ip"]
```

Comment:
```suggestion
            temp = mclag_entry.get("peer_ip")
```

Reply:
@haslersn thanks for suggestions. addressed it